admin/templates/nodes/storage_status.html

```diff
@@ -0,0 +1,11 @@
+{% if resource.storage_limit_status == StorageLimits.DEFAULT %}
```

Comment:
This doesn't appear to be working:

<img width="1194" alt="Screen Shot 2020-09-18 at 11 12 50 AM" src="https://user-images.githubusercontent.com/9688518/93614988-d7461780-f9a0-11ea-8dc4-00db02898ae1.png">

This should obviously be indicating OVER_PUBLIC.